### PR TITLE
Feature / base layout

### PR DESCRIPTION
**Pull request by SedlarDavid**

### CODE REVIEW

File: src/components/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import React from 'react'
```
Comment (tomesfilip) [inline src/components/Header.tsx:1]: no need to import the whole React library since React 16

File: src/components/Page.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
+
+const PageWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100vw`;
+
+// @ts-ignore
+export const Page = (props) => {
```
Comment (tomesfilip) [inline src/components/Page.tsx:9]: the better way how to handle children props is like this:
1. create a type (or interface) before the function
2. use the props type (or interface) as a function's argument type
3. use object destructuring (optional) to take the separate arguments

```
interface Props {
  children: React.ReactNode;
}

const Page = ({children}: Props) => {
return (
  <>
    {children}
  </>
);
}
```

File: src/components/Socials.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    youtube
+}
+
+interface ISocialItem {
+    type: SocialType;
+    link: string;
+    alt: string;
+}
+
+const SocialItems: Array<ISocialItem> = [
```
Comment (tomesfilip) [inline src/components/Socials.tsx:30]: here you can also use a shorter way (optional):
`const SocialItems: ISocialItem[] = [...]`

File: src/components/Socials.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        case SocialType.instagram:
+            return instagram;
+        case SocialType.youtube:
+            return youtube;
+
+    }
+}
+
+export const Socials = () => <SocialsWrapper>
+    {SocialItems.map(s => <a key={s.type} href={s.link}>
```
Comment (tomesfilip) [inline src/components/Socials.tsx:49]: sometimes it's nice to use object destructuring when iterating through the list of objects (also optional):

`SocialItems.map(({type, link, alt}) => (<SocialImage />))`

File: src/components/main/presets/PresetsSection.tsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import React from "react";
```
Comment (tomesfilip) [inline src/components/main/presets/PresetsSection.tsx:1]: don't need to import React lib

File: src/components/main/presets/PresetsWrapper.tsx [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+export const PresetsWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
```
Comment (tomesfilip) [inline src/components/main/presets/PresetsWrapper.tsx:5]: default flex-direction is row, if you remove this line there's no difference (optional..)

File: src/components/main/presets/header/PresetsHeader.tsx [hunk 7]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
```
Comment (tomesfilip) [inline src/components/main/presets/header/PresetsHeader.tsx:1]: no need to import React lib

File: src/components/main/presets/more/PresetsMore.tsx [hunk 8]
```diff
@@ -0,0 +1,13 @@
+import arrow_down from "../../../../assets/icons/arrow_down.svg";
+import React from "react";
```
Comment (tomesfilip) [inline src/components/main/presets/more/PresetsMore.tsx:2]: no need to import React lib

File: src/components/main/presets/preset/Preset.tsx [hunk 9]
```diff
@@ -0,0 +1,14 @@
+import {PresetWrapper} from "./PresetWrapper";
+import React from "react";
```
Comment (tomesfilip) [inline src/components/main/presets/preset/Preset.tsx:2]: no need to import React lib

File: src/components/main/rector/RectorSection.tsx [hunk 10]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
```
Comment (tomesfilip) [inline src/components/main/rector/RectorSection.tsx:1]: no need to import React lib

File: src/components/main/rector/image/RectorImage.tsx [hunk 11]
```diff
@@ -0,0 +1,8 @@
+import rector from "../../../../assets/images/rector.jpg";
+import React from "react";
```
Comment (tomesfilip) [inline src/components/main/rector/image/RectorImage.tsx:2]: no need to import React lib

File: src/components/main/rector/text/RectorText.tsx [hunk 12]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
```
Comment (tomesfilip) [inline src/components/main/rector/text/RectorText.tsx:1]: no need to import React lib

File: src/components/main/splash/Splash.tsx [hunk 13]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
```
Comment (tomesfilip) [inline src/components/main/splash/Splash.tsx:1]: no need to import React lib

File: src/index.css [hunk 14]
```diff
@@ -3,103 +3,39 @@
 @tailwind utilities;
```
Comment (tomesfilip) [inline src/index.css:3]: if tailwind is not used, this can be removed

File: src/pages/Main.tsx [hunk 15]
```diff
@@ -1,14 +1,27 @@
 import React from "react";
```
Comment (tomesfilip) [inline src/pages/Main.tsx:1]: no need to import React lib

**Review by tomesfilip — CHANGES_REQUESTED** [review]
- for files where only styles are applied (styled-components), I would prefer to use the ".styled" suffix (eg. CityPicker.styled.tsx)
- styles should be in a separate file (Header.tsx, ). It's better to have more single purpose files.
- I would avoid using Divider.tsx just for spacing: usually it's better to use margin/padding styles on the elements, or if flexbox is used you can use gap style. depends what's more appropriate for the situation. When using Divider.tsx, you are rendering unnecessary element
- the React lib is imported in the main.tsx file and that is enough for the lib init

**Comment by SedlarDavid** [comment]
Thanks for this review!

**Review by tomesfilip — APPROVED** [review]
gut job gut job 💯

**Pull request merged**